autorest/multiapi/__init__.py

```diff
@@ -0,0 +1,334 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import importlib
+import pkgutil
+import sys
+import logging
+import azure
+import json
+import shutil
+
+from collections import defaultdict
+from pathlib import Path
+from typing import Any, Dict, List, Tuple, Optional, Union
+
+from .multiapi_serializer import MultiAPISerializer
+
+try:
+    import msrestazure
```

Comment:
If you removed all introspection thanks to metadata.json, please remove the msrestazure import and improtlib import